lib/matplotlib/tri/tricontour.py

```diff
@@ -50,11 +50,8 @@ def _process_args(self, *args, **kwargs):
         else:
             tri, z = self._contour_args(args, kwargs)
             C = _tri.TriContourGenerator(tri.get_cpp_triangulation(), z)
-            x0 = tri.x.min()
-            x1 = tri.x.max()
-            y0 = tri.y.min()
-            y1 = tri.y.max()
-            self.ax.update_datalim([(x0, y0), (x1, y1)])
```

Comment:
I do not think we need the line below this either, the autoscaling gets called later in the init.